src/helpers/TokenHelper.js

```diff
@@ -0,0 +1,46 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+/**
+ * @export
+ * @class EmailService
+ */
+class TokenHelper {
+  /**
+   * register a new
+   * @static
+   * @param {Object} user the template to use
+   * @returns {Object} sendEmail
+   */
+  static generateToken(user) {
+    const { id, email } = user;
+    const payload = {
+      id,
+      email
+    };
+    const token = jwt.sign(payload, process.env.JWT_KEY, { expiresIn: '7d' });
+    return token;
+  }
+
+  /**
+   * register a new
+   * @static
+   * @param {Object} token the template to use
+   * @returns {Object} sendEmail
+   */
+  static verifyToken(token) {
+    let userInfo = {};
+    jwt.verify(token, process.env.JWT_KEY, (err, decoded) => {
+      if (err) {
+        userInfo = {
+          error: 'unauthorized token',
+        };
+        return;
+      }
+      userInfo = decoded;
+    });
+    return userInfo;
+  }
+}
+export default TokenHelper;
```

Comment:
I appreciate the way you did this. I was doing another one working similar to this on my new task, You have to implement all the feedback mentioned above and merge it for me to get this too!

Reply:
Okey @william000000